create_framework/http_foundation.rst

```diff
@@ -309,5 +309,5 @@ applications using it (like `Symfony`_, `Drupal 8`_, `phpBB 4`_, `ezPublish
 .. _`Midgard CMS`: http://www.midgard-project.org/
 .. _`Zikula`: http://zikula.org/
 .. _`autoloaded`: http://php.net/autoload
-.. _`PSR-0`: https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-0.md
+.. _`PSR-4`: https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-4-autoloader.md
```

Comment:
Unrelated to this PR, but if you can make this change it would be great: Let's change this URL to the official site: http://www.php-fig.org/psr/psr-4/